### PR TITLE
Added json return for addFile method in file Manager API.

### PR DESCRIPTION
**Pull request by mickel1138**
Added dialog in file manager vue to show renamed files.

### CODE REVIEW

File: assets/js/vue/FileManager.vue [hunk 1]
```diff
@@ -352,6 +371,16 @@ export default {
 
         onHelpButtonClick: function () {
             this.showHelpPopup = true;
+        },
+
+        queueDone: function (message) {
+            if (this.filesRenamed > 0) {
```
Comment (praneethpr) [inline assets/js/vue/FileManager.vue:377]: If this function is to set the isRenamedPopupVisible, we can do it from where it is being called

**Review by praneethpr — COMMENTED** [review]
Tested and works fine

File: assets/js/vue/FileManager.vue (hunk 1 above)
Comment (mickel1138) [reply assets/js/vue/FileManager.vue:377]: You mean like: `this.isRenamedPopupVisible = this.filesRenamed > 0` ??

File: assets/js/vue/FileManager.vue [hunk 2]
```diff
@@ -532,6 +564,7 @@ const initDropzone = () => {
     myDropzone.on("queuecomplete", function () {
         myFileManager.instance.repaint();
         this.removeAllFiles();
+        myFileManager.$parent.queueDone();
```
Comment (praneethpr) [inline assets/js/vue/FileManager.vue:567]:
```suggestion
        myFileManager.$parent.isRenamedPopupVisible = myFileManger.$parent.filesRenamed > 0;
```

File: assets/js/vue/FileManager.vue (hunk 1 above)
Comment (praneethpr) [reply assets/js/vue/FileManager.vue:377]: I put in a code suggestion below

File: assets/js/vue/FileManager.vue [hunk 3]
```diff
@@ -352,6 +371,16 @@ export default {
 
         onHelpButtonClick: function () {
             this.showHelpPopup = true;
+        },
+
+        queueDone: function (message) {
```
Comment (mickel1138) [inline assets/js/vue/FileManager.vue:376]:
```suggestion
        queueDone: function () {
```

File: assets/js/vue/FileManager.vue [hunk 4]
```diff
@@ -352,6 +371,16 @@ export default {
...
+            if (this.filesRenamed > 0) {
+                this.isRenamedPopupVisible = true;
+            }
+        },
```
Comment (mickel1138) [inline assets/js/vue/FileManager.vue:380]:
```suggestion
        
```

File: assets/js/vue/FileManager.vue [hunk 5]
```diff
@@ -347,6 +366,12 @@ export default {
 
         onHelpButtonClick: function () {
             this.showHelpPopup = true;
+        },
+
+
```
Comment (mickel1138) [inline assets/js/vue/FileManager.vue:371]:
```suggestion
```

File: assets/js/vue/FileManager.vue [hunk 6]
```diff
@@ -527,6 +553,7 @@ const initDropzone = () => {
     myDropzone.on("queuecomplete", function () {
         myFileManager.instance.repaint();
         this.removeAllFiles();
+        myFileManager.$parent.isRenamedPopupVisible = myFileManger.$parent.filesRenamed > 0;
```
Comment (praneethpr) [inline assets/js/vue/FileManager.vue:556]:
```suggestion
        myFileManager.$parent.isRenamedPopupVisible = myFileManager.$parent.filesRenamed > 0;
```

**Review by praneethpr — APPROVED** [review]
Tested and works

**Pull request merged**